var/spack/repos/builtin/packages/py-igor/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyIgor(PythonPackage):
+    """igor: interface for reading binary IGOR files."""
+
+    homepage = "http://blog.tremily.us/posts/igor/"
+    pypi = "igor/igor-0.3.tar.gz"
+
+    version("0.3", sha256="b04ac38c68fb81cf3167a9981dc5a20379112d40268bb72c5a8514dc8051abba")
+
+    def patch(self):
+        setupfile = FileFilter("setup.py")
+        setupfile.filter("distutils.core", "setuptools")
```

Comment:
I believe pip does this automatically. Not sure about build which we're planning to switch to in the future.